### PR TITLE
resolves #2028 : work on add heatfluid toolbox python wrapper 

### LINKED ISSUE #2028: add heatfluid toolbox python wrapper
the heatfluid toolbox should now be wrapped in python

### PR DESCRIPTION
**Pull request by prudhomm**
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/feelpp/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes?
- [x] Have you successfully run the Feel++ testsuite with your changes locally?
- [ ] Have you written Doxygen comments in your contribution ?

-----

- closes #2028

### CODE REVIEW

**Comment by prudhomm** [comment]
@thomas-saigre @vincentchabannes the tests for heatfluid and thermoelectric are up and running
let's see what the CI says

**Comment by prudhomm** [comment]
@vincentchabannes the pr is ready

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import sys
+import feelpp
+from feelpp.toolboxes.thermoelectric import *
+e=feelpp.Environment(sys.argv,opts=toolboxes_options("thermo-electric"))
```
Comment (vincentchabannes) [inline python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py]: thermo-electric -> heat-fluid

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+
+f=thermoelectric(dim=2,orderPotential=1,worldComm=e.worldCommPtr())
```
Comment (vincentchabannes) [inline python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py]: heatfluid and modifiy arguments

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/__init__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+except ImportError as e:
+    print('Import feelpp.toolboxes.heatfluid failed: Feel++ Toolbox heatfluid is not available')
+    pass  # module doesn't exist, deal with it.
+
+
+def heatfluid(dim=2, orderTemperature=1, orderVelocity=1, orderPressure=1, worldComm=None, keyword="heatfluid", subprefix="", modelRep=None):
+    """create a heatfluid toolbox solver
+    Keyword arguments:
+    dim -- the dimension (default: 2)
+    orderPotential -- the polynomial order for the potential (default: 1)
```
Comment (vincentchabannes) [inline python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/__init__.py]: fix copy past for heatfluid

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.cpp [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    using element_temperature_t = typename toolbox_t::heat_model_type::element_temperature_type;
+    using element_temperature_ptr_t = typename toolbox_t::heat_model_type::element_temperature_ptrtype;
+//    using element_heatfluidpotential_ptr_t = typename toolbox_t::electric_model_type::element_electricpotential_ptrtype;
+
+    std::string pyclass_name = fmt::format("HeatFluid_{}D_P{}_P{}P{}",nDim,OrderT,OrderV,OrderP);
+
+    py::class_<toolbox_t,std::shared_ptr<toolbox_t>,ModelNumerical>(m,pyclass_name.c_str())
+        .def(py::init<std::string const&,std::string const&,worldcomm_ptr_t const&,std::string const&, ModelBaseRepository const&>(),
+             py::arg("prefix"),
+             py::arg("keyword")=std::string("thermo-electric"),
```
Comment (vincentchabannes) [inline python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.cpp]: heat-fluid

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py (hunk 2 above)
Comment (prudhomm) [reply python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py]: I will remove the code `heatfluid.py` it should not be there.

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py (hunk 1 above)
Comment (prudhomm) [reply python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.py]: I will remove the code heatfluid.py it should not be there.

File: python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.cpp (hunk 4 above)
Comment (prudhomm) [reply python/pyfeelpp-toolboxes/feelpp/toolboxes/heatfluid/heatfluid.cpp]: it is done properly (passing keyword as `heat-fluid` by default in `__init__,py` but indeed this is not nice

**Pull request merged**